reconcile.sh

```diff
@@ -2,9 +2,15 @@
 
 oc delete workflow $(oc get workflows -o=jsonpath='{.items[?(@.status.phase=="Succeeded")].metadata.name}')
 
-for w in $( oc get workflows -o=jsonpath='{.items[?(@.status.phase=="Error")].metadata.name}' ); do
-    argo retry $w
-    sleep 7
+while :
+do
+    for w in $(oc get workflows -o=jsonpath='{.items[?(@.status.phase=="Error")].metadata.name}' ); do
+        argo retry $w
+        sleep 7
+        oc delete workflow $(oc get workflows -o=jsonpath='{.items[?(@.status.phase=="Succeeded")].metadata.name}')
```

Comment:
By this, we're saying that the whole logic is encapsulated in the Workflow and there is no other component interacting with it that is not part of the Workflow. Is that a valid assumption?

Reply:
for a solver workflow... i think that is true

Reply:
@fridex

Reply:
Once we have data on ceph and the sync to the graph is done, we don't need openshift objects anymore.